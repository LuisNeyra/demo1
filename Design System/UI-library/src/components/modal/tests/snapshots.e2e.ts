import { snapIt } from '../../../test';

const component = 'modal';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap('renders correctly with no props', '<blaze-modal>default</blaze-modal>');
    snap('renders as open', '<blaze-modal open>default</blaze-modal>');
    snap('renders dismissible button and overlay', '<blaze-modal open dismissible>default</blaze-modal>');
  });
});
