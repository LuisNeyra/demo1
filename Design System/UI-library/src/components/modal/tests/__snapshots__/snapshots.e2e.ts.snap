// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`modal renders renders as open 1`] = `"<blaze-modal open=\\"\\" class=\\"hydrated\\"><!----><div aria-hidden=\\"\\" class=\\"c-overlay c-overlay--fullpage c-overlay--visible\\"></div><div role=\\"dialog\\" class=\\"o-modal o-modal--visible\\">default</div></blaze-modal>"`;

exports[`modal renders renders correctly with no props 1`] = `"<blaze-modal class=\\"hydrated\\"><!----><div aria-hidden=\\"\\" class=\\"c-overlay c-overlay--fullpage\\"></div><div role=\\"dialog\\" class=\\"o-modal\\">default</div></blaze-modal>"`;

exports[`modal renders renders dismissible button and overlay 1`] = `"<blaze-modal open=\\"\\" dismissible=\\"\\" class=\\"hydrated\\"><!----><div aria-hidden=\\"\\" class=\\"c-overlay c-overlay--fullpage c-overlay--visible\\"></div><div role=\\"dialog\\" class=\\"o-modal o-modal--visible\\"><button type=\\"button\\" class=\\"c-button c-button--close\\">Ã—</button>default</div></blaze-modal>"`;
